torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1400,13 +1400,8 @@ def create_scalar():
         lifted_args = tuple(arg for arg in body_lifted_freevars.keys())
 
         proxy_args = (body_node,) + lifted_args
-        example_value = pytree.tree_map_only(
```

Comment:
can you roll the dtype changes into your PR btw?

Reply:
changed the scalars to float32